src/aind_data_transfer/transformations/metadata_creation.py

```diff
@@ -140,14 +164,16 @@ def ephys_job_to_subject(
 class DataDescriptionMetadata:
     """Class to handle the creation of the processing metadata file."""
 
+    output_file_name = RawDataDescription.construct()._get_default_filename()
+
     @staticmethod
-    def ephys_job_to_data_description(
+    def get_data_description(
```

Comment:
Can we expose funding to a config file in case someone from AIBS wants to use this script? Happy to take that as a separate issue.

Reply:
Yeah, maybe we can push it off to another ticket? I think at this point, we might want to prioritize creating a basic gui to facilitate configuring the jobs.